package/MDAnalysis/analysis/backends.py

```diff
@@ -0,0 +1,182 @@
+"""Analysis backends --- :mod:`MDAnalysis.analysis.backends`
+
+Module introduces :class:`BackendBase` base class to implement custom
+backends for :meth:`MDAnalysis.analysis.base.AnalysisBase.run()` and its
+subclasses. Also, it introduces 2 built-in backend classes:
+:class:`BackendMultiprocessing` that supports parallelization via standard
+python ``multiprocessing`` module, and :class:`BackendDask`, that uses the same
+process-based parallelization as :class:`BackendMultiprocessing`, but different
+serialization algorithm.
```

Comment:
I'd probably expand the description here, to clarify the different serialization.

Reply:
Reference to the documentation sounds perfect, thanks!